### PR TITLE
Uplink endpoint env variable name

### PR DESCRIPTION
**Pull request by BrynCooke**
The proposal is that we move to the following on the Router.
* `APOLLO_SCHEMA_CONFIG_DELIVERY_ENDPOINT` -> `APOLLO_UPLINK_ENDPOINTS`
* `--apollo-schema-delivery-endpointl` -> `--apollo-uplink-endpoints`

And that we follow up by adding `APOLLO_UPLINK_ENDPOINTS` as an env variable on the gateway. But NOT  removing `APOLLO_SCHEMA_CONFIG_DELIVERY_ENDPOINT`.

Rationale:
* `APOLLO_UPLINK_ENDPOINTS` aligns with the gateway `ManagedGatewayConfig#uplinkEndpoints`.  
* It's a better name that aligns with our product name.
* There's enough differences with the router config that we should think hard if users will benefit from keeping the old name.

### CODE REVIEW

**Comment by abernix** [comment]
@glasser Is this something you can opine on?

**Review by bnjjj — APPROVED** [review]

File: apollo-router/src/executable.rs [hunk 1]
```diff
@@ -69,7 +69,7 @@ pub struct Opt {
 
     /// The endpoint polled to fetch the latest supergraph schema.
     #[clap(long, env)]
-    apollo_uplink_url: Option<Url>,
+    apollo_uplink_endpoints: Option<Url>,
```
Comment (glasser) [inline apollo-router/src/executable.rs:72]: I assume this actually affects the env var used via the `env` on the previous line?

**Review by glasser — COMMENTED** [review]
This looks good to me, assuming that the proposal to add this to Gateway happens too.  But @trevor-scheer would know this history better than me.

File: apollo-router/src/executable.rs (hunk 1 above)
Comment (BrynCooke) [reply apollo-router/src/executable.rs:72]: It does!

**Pull request merged**